static/app/views/replays/detail/focusArea.tsx

```diff
@@ -55,34 +86,15 @@ function FocusArea({}: Props) {
         />
       );
     case 'network': {
-      // Fake the span and Trace context
-      const nonMemorySpansEntry = {
-        type: EntryType.SPANS,
-        data: replay
-          .getRawSpans()
-          .filter(replay.isNotMemorySpan)
-          .map(({startTimestamp, endTimestamp, ...span}) => ({
-            ...span,
-            timestamp: endTimestamp,
-            start_timestamp: startTimestamp,
-            span_id: uuid4(), // TODO(replays): used as a React key
-            parent_span_id: 'replay_network_trace',
-          })),
-      };
-      const performanceEvent = {
-        ...event,
-        contexts: {
-          trace: {
-            type: 'trace',
-            op: 'Network',
-            description: 'WIP',
-            span_id: 'replay_network_trace',
-            status: 'ok',
-          },
-        },
-        entries: [nonMemorySpansEntry],
-      } as EventTransaction;
-      return <Spans organization={organization} event={performanceEvent} />;
+      return <Spans organization={organization} event={getPerformanceEvents()} />;
+    }
+    case 'network 2': {
```

Comment:
Nit: could add curlies to make the console case a block: `case 'console': { ... }`, and remove from the other two.
Curlies `{}` create a block, which creates scope for variables. So it's useful only when there are variables inside afaik.